content/methods/interview-checklist.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Interview checklist
+permalink: /methods/interview-checklist/
+layout: layouts/page
+---
```

Comment:
@jduss4 [blocking request] Let's add the content for this page while we're at it.

Reply:
Can do!   Some of these pages link to other pages which link to other pages (including other categories) which is why I had held off initially.  I'll see how it goes when I get into it.

Reply:
After discussing, we'll handle this in another PR